packages/venia-ui/lib/components/Head/headProvider.js

```diff
@@ -1,19 +1,12 @@
-import React, { useEffect } from 'react';
-import { HeadProvider } from 'react-head';
-
-const removeExistingTitleTags = titleTags => {
-    titleTags.forEach(titleTag => {
-        if (titleTag) titleTag.parentNode.removeChild(titleTag);
-    });
-};
+import React from 'react';
+import {HelmetProvider as HeadProvider} from 'react-helmet-async';
 
 const VeniaHeadProvider = props => {
-    useEffect(() => {
-        const titleTagsToRemove = document.getElementsByTagName('title');
-        removeExistingTitleTags(Array.from(titleTagsToRemove));
-    }, []);
-
-    return <HeadProvider>{props.children}</HeadProvider>;
+    if (VeniaHeadProvider.canUseDom !== null) {
+        HeadProvider.canUseDOM = VeniaHeadProvider.canUseDom;
+    }
```

Comment:
Is `canUseDOM` purely for testing?

If so, we should remove all code related to it from this file and instead handle it it by mocking `HeadProvider` in the tests.

Reply:
I did add it during yes. Seems it would only be needed if testing SSR and Jest - which we aren't doing yet (for now). Will remove it.